client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/screen-confirmation.tsx

```diff
@@ -1,22 +1,44 @@
 import { Button } from '@automattic/components';
+import { Design } from '@automattic/design-picker';
 import { NavigatorHeader } from '@automattic/onboarding';
 import {
 	__experimentalVStack as VStack,
 	__experimentalHStack as HStack,
 } from '@wordpress/components';
 import { Icon, image, verse, layout } from '@wordpress/icons';
 import { useTranslate } from 'i18n-calypso';
+import { useSelector } from 'react-redux';
+import { getActiveTheme } from 'calypso/state/themes/selectors';
+import { IAppState } from 'calypso/state/types';
```

Comment:
[nit] 🙂
```suggestion
import type { IAppState } from 'calypso/state/types';
```

Reply:
Stumped that it's not handled by our autoformatter T_T